db/fixtures/classifications.yml

```diff
@@ -1202,3 +1202,323 @@
   :parent_id: 0
   :default: true
   :single_value: "1"
+- :description: Quota - Warn CPUs
+  :entries:
+  - :description: "1"
+    :read_only: "0"
+    :syntax: string
+    :name: "1"
+    :example_text:
+    :default: true
+    :single_value: "1"
+  - :description: "2"
+    :read_only: "0"
+    :syntax: string
+    :name: "2"
+    :example_text:
+    :default: true
+    :single_value: "1"
+  - :description: "3"
+    :read_only: "0"
+    :syntax: string
+    :name: "3"
+    :example_text:
+    :default: true
+    :single_value: "1"
+  - :description: "4"
+    :read_only: "0"
+    :syntax: string
+    :name: "4"
+    :example_text:
+    :default: true
+    :single_value: "1"
+  - :description: "5"
+    :read_only: "0"
+    :syntax: string
+    :name: "5"
+    :example_text:
+    :default: true
+    :single_value: "1"
+  - :description: "10"
+    :read_only: "0"
+    :syntax: string
+    :name: "10"
+    :example_text:
+    :default: true
+    :single_value: "1"
+  - :description: "20"
+    :read_only: "0"
+    :syntax: string
+    :name: "20"
+    :example_text:
+    :default: true
+    :single_value: "1"
+  - :description: "30"
+    :read_only: "0"
+    :syntax: string
+    :name: "30"
+    :example_text:
+    :default: true
+    :single_value: "1"
+  - :description: "40"
+    :read_only: "0"
+    :syntax: string
+    :name: "40"
+    :example_text:
+    :default: true
+    :single_value: "1"
+  - :description: "50"
+    :read_only: "0"
+    :syntax: string
+    :name: "50"
+    :example_text:
+    :default: true
+    :single_value: "1"
+  :read_only: "0"
+  :syntax: string
+  :show: true
+  :name: quota_warn_cpu
+  :example_text: Warning number of CPUs allowed by Quota
+  :parent_id: 0
+  :default: true
+  :single_value: "1"
+- :description: Quota - Warn  Memory
+  :entries:
+  - :description: 1GB
+    :read_only: "0"
+    :syntax: string
+    :name: "1024"
```

Comment:
Why is the name not set the same way as for Quota - Warn  Storage?

Reply:
@lfu Unfortunately, tags values differ for storage and memory.  The tag values from max_storage/warn_storage should match, as well as values for max_memory/warn_memory.

Reply:
We can't change the values because it would break existing customers environments.